python/dataproc_templates/cassandra/README.md

```diff
@@ -61,3 +65,23 @@ export REGION=<region>
 	--cassandratobq.temp.gcs.location=<staging bucket name> \
 	--cassandratobq.output.mode=<overwrite|append|ignore|errorifexists>
 ```
+## Sample Submission
+After downloading the jar, a sample submission -:
+```
+export GCP_PROJECT=my-project
+export REGION=us-central1
+export GCS_STAGING_LOCATION=gs://staging_bucket
+export JOB_TYPE=SERVERLESS
+export SUBNET=projects/my-project/regions/us-central1/subnetworks/default
+export JARS="spark-cassandra-connector-assembly_2.12-3.2.0.jar,gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar,jnr-posix-3.1.8.jar"
+
+
+    ./bin/start.sh \
+-- --template=CASSANDRATOBQ \
```

Comment:
add dummy values for sensitive data like IP

Reply:
the provided IP is a dummy IP. This is done on purpose because since we had similar requirements in other templates to show working example below the structure example

Reply:
I will add redaction for further security